### DISCUSSION TITLE
log user information of workflows started via GUI

### DISCUSSION THREAD

**Discussion by florian-feicht** [argoproj/argo-workflows#12238] [Q&A] [answered]
Hi all,
we have a SSO configuration in place for our ArgoWF setup.
For compliance reasons, we would need to log the information about the logged in user for each workflow execution. 
Is anyone aware of a way how to handle this requirement?
Regards,
Florian

**Comment by agilgur5** [comment, marked as answer]
The [`workflows.argoproj.io/creator` labels](https://argo-workflows.readthedocs.io/en/latest/workflow-creator/) should provide this information when submitted through the UI with a logged in user.
